assets/console-plugin/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+---
+# Source: openshift-console-plugin/templates/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: monitoring-plugin
+  namespace: openshift-monitoring
+  labels:
+    app: monitoring-plugin
+    app.kubernetes.io/name: monitoring-plugin
+    app.kubernetes.io/instance: monitoring-plugin
+    app.kubernetes.io/part-of: monitoring-plugin
+    app.openshift.io/runtime-namespace: monitoring-plugin
+spec:
```

Comment:
We lack anti-affinity rules + pod disruption budget to prevent service disruption in case of node upgrades.

Reply:
Good point! . I will add those.

Reply:
Done, could you please check again?